torch/_higher_order_ops/effects.py

```diff
@@ -196,11 +198,16 @@ def with_effects_proxy(
 with_effects.fallthrough(DispatchKey.AutogradCUDA)
 
 
-def _get_schema(op, args) -> torch.FunctionSchema:
+def _get_schema(op, args, kwargs: Optional[dict] = None) -> torch.FunctionSchema:
     if isinstance(op, torch._ops.OpOverload):
         return op._schema
     elif op == call_torchbind:
         return getattr(args[0], args[1]).schema
+    elif op == hop_print:
+        # hop_print expects (format_str, *args) as its arguments
+        format_str = args[0]
+        extra_kwargs = kwargs if kwargs is not None else {}
+        return op.gen_schema(format_str, **extra_kwargs)
```

Comment:
you should just be able to pass in op.gen_schema(*args, **kwargs)

Reply:
thanks. good python learning here.